PSL/SENSORS/Sx1276.py

```diff
@@ -0,0 +1,331 @@
+#Registers adapted from sample code for SEMTECH SX1276
+import time
+
+def connect(SPI,frq,**kwargs):
+	return SX1276(SPI,frq,**kwargs)
+
+
+class SX1276():
+	name = 'SX1276'
+	#registers
+	REG_FIFO                 = 0x00
+	REG_OP_MODE              = 0x01
+	REG_FRF_MSB              = 0x06
+	REG_FRF_MID              = 0x07
+	REG_FRF_LSB              = 0x08
+	REG_PA_CONFIG            = 0x09
+	REG_LNA                  = 0x0c
+	REG_FIFO_ADDR_PTR        = 0x0d
+	REG_FIFO_TX_BASE_ADDR    = 0x0e
+	REG_FIFO_RX_BASE_ADDR    = 0x0f
+	REG_FIFO_RX_CURRENT_ADDR = 0x10
+	REG_IRQ_FLAGS            = 0x12
+	REG_RX_NB_BYTES          = 0x13
+	REG_PKT_RSSI_VALUE       = 0x1a
+	REG_PKT_SNR_VALUE        = 0x1b
+	REG_MODEM_CONFIG_1       = 0x1d
+	REG_MODEM_CONFIG_2       = 0x1e
+	REG_PREAMBLE_MSB         = 0x20
+	REG_PREAMBLE_LSB         = 0x21
+	REG_PAYLOAD_LENGTH       = 0x22
+	REG_MODEM_CONFIG_3       = 0x26
+	REG_RSSI_WIDEBAND        = 0x2c
+	REG_DETECTION_OPTIMIZE   = 0x31
+	REG_DETECTION_THRESHOLD  = 0x37
+	REG_SYNC_WORD            = 0x39
+	REG_DIO_MAPPING_1        = 0x40
+	REG_VERSION              = 0x42
+	REG_PA_DAC				 = 0x4D
```

Comment:
As long as the underlying chip is SX1276, this library will work. I've used the 434 MHz modules from Dorji (DRF1276) as well as the RA-02 from AI-thinker. There are 868 MHz versions also, and only the right frequency needs to be selected in the code.